.drone.starlark

```diff
@@ -394,6 +431,52 @@ config = {
 				}
 			],
 		},
+		'core-api-federation-latest-nightly': {
+			'suites': [
+				'apiFederation',
+			],
+			'databases': [
+				'mysql:5.7',
+			],
+			'servers': [
+				'latest',
```

Comment:
compared to the section above, this new section runs against `latest` rather than `daily-master-qa`